diff --git a/node_modules/react-scrollspy/lib/scrollspy.js b/node_modules/react-scrollspy/lib/scrollspy.js
index 4caf0b3..9780ce7 100644
--- a/node_modules/react-scrollspy/lib/scrollspy.js
+++ b/node_modules/react-scrollspy/lib/scrollspy.js
@@ -51,7 +51,7 @@ function (_React$Component) {
         currentClassName: _propTypes.default.string.isRequired,
         scrolledPastClassName: _propTypes.default.string,
         style: _propTypes.default.object,
-        componentTag: _propTypes.default.oneOfType([_propTypes.default.string, _propTypes.default.element]),
+        componentTag: _propTypes.default.oneOfType([_propTypes.default.string, _propTypes.default.elementType]),
         offset: _propTypes.default.number,
         rootEl: _propTypes.default.string,
         onUpdate: _propTypes.default.func
